### ISSUE TITLE
Which loss value I should stop pre-trainning?

### ISSUE THREAD

**Issue by ngoanpv** [brightmart/albert_zh#70]
Hi, 
I am confused how to know time to stop pre-training, please tell me? 
Because I see the loss value is not always decrease  in the step that you stop.

**Comment by brightmart** [comment]
me too. you can  spilt train and dev corpus before pretrain. when pretrain, you can check from time to time the accuracy of mask lon dev set. if acc increase it is learning, you can find code in pretrain py fil
